decidim-multisurveys/app/forms/decidim/admin/multisurvey_form.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class MultisurveyForm < Form
```

Comment:
The form which takes input from the POST request to create or update a Multisurvey